docs/_static/css/htcondor-manual.css

```diff
@@ -36,33 +36,40 @@ div.wy-side-nav-search a.icon.icon-home:before {
 	color: #d31245;
 }
 
-/* Override inline literal (``some text``) */
-.rst-content code.literal,.rst-content tt.literal {
-	text-decoration: underline #d3d3d3 1px;
-	font-weight: 575;
-	font-size: 85%;
-	font-style: italic;
-	/*color: #8B0000; Dark Red*/
-	color: #000000;
-	background-color: transparent;
-	border: 0px solid #e1e1e1;
-	padding: 1px 1px;
-	white-space: normal;
-}
-
 /* override table width restrictions */
 @media screen and (min-width: 767px) {
-
 	.wy-table-responsive table td {
 	   /* !important prevents the common CSS stylesheets from overriding
 		  this as on RTD they are loaded after this stylesheet */
 	   white-space: normal !important;
 	}
- 
+
 	.wy-table-responsive {
 	   overflow: visible !important;
 	}
+
 	.wy-nav-content {
 	    max-width: 95%;
 	}
  }
+
+/* override the formatting of Python documentation links */
+.rst-content code.xref.py.py-obj.docutils.literal.notranslate,
+.rst-content code.xref.py.py-data.docutils.literal.notranslate,
+.rst-content code.xref.py.py-meth.docutils.literal.notranslate,
+.rst-content code.xref.py.py-class.docutils.literal.notranslate
+{
+    color: #000000;
+    text-decoration: underline;
+    border: none;
+    background-color: transparent;
+    font-weight: normal;
+}
+
+/* override the ``literal`` formating */
+.rst-content code:not(.xref):not(.py):not(.py-data).docutils.literal.notranslate,
+.rst-content   tt:not(.xref):not(.py):not(.py-data).docutils.literal.notranslate
+{
+    color: #000000;
+    background-color: transparent;
+}
```

Comment:
I think this restores the box around literals.  Is there a better way to set off literals other than the box?  At least the text isn't red, which was awful

Reply:
At least on my screens, the monospace font doesn't set the literals off by as much as I would like.  I've added a background to see how that works: I don't want to use bold (it's used for all sorts of other purposes in the manual, or at least the Python API docs, already) and I don't want to use italic (because that normally means the opposite of literal in documentation, and it's already used (or should be) in the descriptive text when referring to parameter names).  Underline is right out (these aren't links), which leaves borders (which you don't like) and color (which you don't like applied to the text).